src/ipmi.c

```diff
@@ -551,13 +783,19 @@ static int c_ipmi_config(const char *key, const char *value) {
     ignorelist_set_invert(ignorelist, invert);
   } else if (strcasecmp("NotifySensorAdd", key) == 0) {
     if (IS_TRUE(value))
-      c_ipmi_nofiy_add = 1;
+      c_ipmi_notify_add = 1;
   } else if (strcasecmp("NotifySensorRemove", key) == 0) {
     if (IS_TRUE(value))
-      c_ipmi_nofiy_remove = 1;
+      c_ipmi_notify_remove = 1;
   } else if (strcasecmp("NotifySensorNotPresent", key) == 0) {
     if (IS_TRUE(value))
-      c_ipmi_nofiy_notpresent = 1;
+      c_ipmi_notify_notpresent = 1;
+  } else if (strcasecmp("SELEnabled", key) == 0) {
+    if (IS_TRUE(value))
```

Comment:
Please use `c_ipmi_sel_enabled = IS_TRUE(value);` here and elsewhere. If a user ends up with a config like this:

```
SELEnabled true
SELEnabled false
```

the second statement should be able to overwrite the previous statement. I'd be very grateful if you could also fix the existing options, thank you!